src/app/common/navbar/navbar.component.html

```diff
@@ -1,5 +1,5 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-primary">
-	<a class="navbar-brand" href="#">Allure reporter</a>
+	<a class="navbar-brand" href="#">Allure reporter{{isLogged}}</a>
```

Comment:
This change may be just for testing. Please, remove it.

Reply:
Ups! ok, thank you